src/main/java/com/github/kiulian/downloader/parser/DefaultParser.java

```diff
@@ -59,7 +59,16 @@ public JSONObject getPlayerConfig(String htmlUrl) throws YoutubeException {
 
         String ytPlayerConfig = extractor.extractYtPlayerConfig(html);
         try {
-            return JSON.parseObject(ytPlayerConfig);
+            JSONObject config = JSON.parseObject(ytPlayerConfig);
+            if(config.containsKey("args")) {
+                return config;
+            } else {
+                JSONObject args = new JSONObject();
+                JSONObject playerResponse = new JSONObject();
+                playerResponse.put("player_response", config);
+                args.put("args", playerResponse);
+                return args;
```

Comment:
Counterintuitive namings of `playerResponse`, `args` objects and keys. Why "player_reponse" is inside `playerResponse` instead of beeing inside `args` as it is in default ytPlayerConfig? And method `getPlayerConfig` should return `config`, but returns `args` in else block. 

Maybe replace with:
```java
JSONObject newConfig = new JSONObject();
JSONObject args = new JSONObject();
args.put("player_response", config);
newConfig.put("args", args);
return newConfig;
// or
return new JSONObject().fluentPut("args", new JSONObject().fluentPut("player_response", config))'
```

Reply:
I wrote it simply for quick modification.
It looks good to write in a line.
```java
return new JSONObject().fluentPut("args", new JSONObject().fluentPut("player_response", config))
```
Please update it after modifying it with this code. 😀

Reply:
@kangsLee commit this line to your branch and I will merge PR, thank you

Reply:
@sealedtx Committed. Thank you.